executor/runtime/types/sidecars.go

```diff
@@ -46,4 +47,10 @@ var sideCars = []SidecarContainerConfig{
 			"/titus/spectatord": {},
 		},
 	},
+	{
+		ServiceName: SidecarServiceAtlasd,
+		Volumes: map[string]struct{}{
+			"/titus/atlas-titus-agent": {},
```

Comment:
for consistency can this be as simple as `/titus/atlasd` ?

Reply:
Depends on the image. Current image is published with that path.

Reply:
The new image will be published with `/titus/atlas-titus-agent` as the path.

As mentioned elsewhere, we have a separate service that performs a different function, which is already named `atlasd`.